src/foam/lib/csv/FObjectCSVOutputterDecorator.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.lib.csv',
+  name: 'FObjectCSVOutputterDecorator',
```

Comment:
Create a ProxyCSVOutputter and make this extend that. This way, we can add methods to the interface and nothing else should need to get outputted.

Reply:
Also rename this to PrefixedValueCSVOutputter ?